src/stream_base.cc

```diff
@@ -292,17 +292,16 @@ void StreamBase::CallJSOnreadMethod(ssize_t nread,
                                     size_t offset) {
   Environment* env = env_;
 
-#ifdef DEBUG
-  CHECK_EQ(static_cast<int32_t>(nread), nread);
-  CHECK_LE(offset, INT32_MAX);
+  DCHECK_EQ(static_cast<int32_t>(nread), nread);
+  DCHECK_LE(offset, INT32_MAX);
 
   if (ab.IsEmpty()) {
```

Comment:
hmm, wait, these branches are now no longer under the `ifdef DEBUG` guard, can you use `DCHECK_IMPLIES`? e.g. see https://github.com/nodejs/node/pull/24563/files